index.js

```diff
@@ -58,8 +182,11 @@ export class Placement {
 
     this.response = null;
 
-    this.keywords = keywords || "";
-    this.campaign_types = campaign_types || "paid|community|house";
+    this.keywords = (keywords || []).concat(this.detectKeywords());
```

Comment:
It might be useful on the server-side to have the user-defined & automatic keywords separate in the request. Maybe we don't need it for a v1 given that it requires server changes (eg. `automatic_keywords` alongside `keywords`), but I do think we might want to treat them differently in the future.

Reply:
Ya, I could see that. It's possible also that this client side stuff is only temporary too. We'll have to see.